src/Fixer/LanguageConstruct/DeclareBracesFixer.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\LanguageConstruct;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Tokens;
+
+final class DeclareBracesFixer extends AbstractFixer
+{
+    public function getDefinition(): FixerDefinitionInterface
+    {
+        return new FixerDefinition(
+            'There must not be spaces around `declare` statement braces.',
```

Comment:
what about this case?
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5751/files#diff-0c70c3fd792f24b02dd316b0fa21ed73dc2b9bbb0fc845a8877c22ae1003dc4cL892

or you want to have 2 fixers around `declare` keyword, one for whitespace around `(` and one for `{` ?